news/_posts/2019-09-10-the-review-process.md

```diff
@@ -0,0 +1,89 @@
+date:   2019-09-10 10:30:00
+authors: [ MathieuFerment ]
+icon: icon-code
+tags:
+ - community
+ - contribution
+ - review
+---
+
+
+The PrestaShop project currently has 389 pull requests open and new Pull Requests are opened almost everyday. And this is only for the main repository. Other github repositories like the [developer documentation](https://github.com/PrestaShop/docs), modules, [docker images](https://github.com/PrestaShop/docker) also have their share of pull requests waiting to be merged.
+
+All of these pull requests go through a thorough process which aims to provide a stable, consistent and reliable software that we all know under the name PrestaShop. Here is this process in details.
+
+## What happens when you open a Pull Request
+
+When you submit a new Pull Request to the project repository https://github.com/PrestaShop/PrestaShop, a long journey starts.
+
+### Continuous Integration
+
+We have setup on the project 2 great tools that help us check the quality of contributions.
+
+The first tool is [PrettyCI](https://prettyci.com/). This tool will look at your Pull Request and check whether the code-style is correct. If not, it will block the PR from being merged and tell you what is wrong.
+
+It might appear trivial, but having a unified code-style on the project actually helps a lot. It is a lot easier to work on a big codebase like PrestaShop if all code is written following the same conventions: snake_case or camelCase, how to write the phpDoc, when to use white spaces ... it makes the code look like if it was written by a single developer. Just like reading a book with two different styles in it, having a codebase with different styles is making it harder to navigate. It also makes the pull requests easier to review !
+
+If PrettyCI states that your pull request has issues, you need to fix the issues by looking at PrettyCI output to understand what needs to be fixed.
+
+The second tool is [Travis](http://travis-ci.org/). Travis is a Continous Integration system that will look at the Pull Request and run several checks, be it code-style checkers, format checkers or automated tests, and provide us the result in the Pull Request. If something is wrong it will block the PR from being merged. This is a standard approach to ensure that new contributions in a codebase do not break existing features and behaviors.
+
+If Travis states that your pull request has issues, you need to fix the issues by looking at Travis output to understand what needs to be fixed.
+
+These 2 tools are being run for every Pull Request, so we know that every contribution has been checked. This is a great asset to help us ensure the quality of the project.
+
+Note: we have also recently activated on the `develop` branch the new [Github Actions](https://github.com/features/actions) and benefit from an additional php7.2-compliancy check.
+
+![CI tools on github](/assets/images/2019/09/CI-github-tools.png)
```

Comment:
Image is in PR https://github.com/PrestaShop/prestashop.github.io/pull/576

Reply:
BTW the easiest way to add image into the same PR so we can see them in the preview, is to commit them on the same branch